index.bs

```diff
@@ -2443,74 +2325,116 @@ partial interface MLGraphBuilder {
-            1. If that [=exception/throws=] an error, then re-[=exception/throw=] the error.
-        1. Return |output|.
-    </div>
+#### equal #### {#api-mlgraphbuilder-logical-equal}
+Compare if the values of the two input tensors are equal, element-wise. See [[#api-mlgraphbuilder-logical]] for more detail.
 
-    <div algorithm>
-    The <dfn method for=MLGraphBuilder>div(|a|, |b|)</dfn> method steps are:
-        1. Let |output| be the result of running the [=MLGraphBuilder/element-wise-binary-op | create element-wise binary operation=] given "div", |a| and |b|.
-            1. If that [=exception/throws=] an error, then re-[=exception/throw=] the error.
-        1. Return |output|.
-    </div>
+#### greater #### {#api-mlgraphbuilder-logical-greater}
+Compare if the values of the first input tensor is greater, element-wise. See [[#api-mlgraphbuilder-logical]] for more detail.
 
-    <div algorithm>
-    The <dfn method for=MLGraphBuilder>max(|a|, |b|)</dfn> method steps are:
-        1. Let |output| be the result of running the [=MLGraphBuilder/element-wise-binary-op | create element-wise binary operation=] given "max", |a| and |b|.
-            1. If that [=exception/throws=] an error, then re-[=exception/throw=] the error.
-        1. Return |output|.
-    </div>
+#### greaterOrEqual #### {#api-mlgraphbuilder-logical-greater-or-equal}
+Compare if the values of the first input tensor is greater or equal, element-wise. See [[#api-mlgraphbuilder-logical]] for more detail.
 
-    <div algorithm>
-    The <dfn method for=MLGraphBuilder>min(|a|, |b|)</dfn> method steps are:
-        1. Let |output| be the result of running the [=MLGraphBuilder/element-wise-binary-op | create element-wise binary operation=] given "min", |a| and |b|.
-            1. If that [=exception/throws=] an error, then re-[=exception/throw=] the error.
-        1. Return |output|.
-    </div>
+#### lesser #### {#api-mlgraphbuilder-logical-lesser}
+Compare if the values of the first input tensor is lesser, element-wise. See [[#api-mlgraphbuilder-logical]] for more detail.
 
-    <div algorithm>
-    The <dfn method for=MLGraphBuilder>pow(|a|, |b|)</dfn> method steps are:
-        1. Let |output| be the result of running the [=MLGraphBuilder/element-wise-binary-op | create element-wise binary operation=] given "pow", |a| and |b|.
-            1. If that [=exception/throws=] an error, then re-[=exception/throw=] the error.
-        1. Return |output|.
-    </div>
-</div>
-</details>
+#### lesserOrEqual #### {#api-mlgraphbuilder-logical-lesser-or-equal}
+Compare if the values of the first input tensor is lesser or equal, element-wise. See [[#api-mlgraphbuilder-logical]] for more detail.
+
+#### not #### {#api-mlgraphbuilder-logical-not}
+Invert the values of the input tensor to Boolean values 0 or 1, element-wise. See [[#api-mlgraphbuilder-logical]] for more detail.
```

Comment:
Define "invert" over the input value range.

Reply:
Fixed.

Reply:
Fixed in https://github.com/webmachinelearning/webnn/pull/478/commits/873eec1e656253ed5e5f600f0ebf7e5fd12b9746